### ISSUE TITLE
Kann kein "Global Objective" setzen

### ISSUE THREAD

**Issue by Enel300** [StrangeLoopGames/EcoIssues#24937]
User Id: (Not logged user)
Version: 0.10.2.3 beta release-669

Steps to Reproduce:
Ernenne dich zum World Leader mit den Rechten "Can Set Global Objective". Klicke auf "O" und versuche ein "Global Objective" festzulegen. 

Expected behavior:
Du hast eine Schaltfläche in der du ein Global Objective festlegen kannst. 

Actual behavior:
Ich sehe unter "O" nur "Your Objective". Wo kann ich Global Objectives festlegen?

Do you have mods installed? Does the issue happen when no mods are installed?:
Nein, Ja

![2024-05-14 23_44_25](https://github.com/StrangeLoopGames/EcoIssues/assets/106102715/d58c0249-7959-4364-9b30-cfb49c622240)

![2024-05-14 23_45_10](https://github.com/StrangeLoopGames/EcoIssues/assets/106102715/5bcfd23c-1ef7-4a83-9ba1-97007272e89a)

**Comment by D3nnis3n** [comment]
Die Funktion "Globales Objective" existiert nicht, wenn das Settlement System aktiviert ist - Tooltips von Titeln zeigen es trotzdem, aber es kann auch nicht angepasst werden, ist das System nicht aktiviert?

**Comment by D3nnis3n** [comment]
Ja, das ist der Fall. Da wir das nicht mehr unterstützen kann ich dir leider nicht versprechen, dass der Fehler (zeitnah) behoben wird. Vermutlich werden wir das Globale Objective ganz entfernen, da das in der UI einfach nicht mehr da ist.

**Comment by Enel300** [comment]
Naja aber du sagst weiter oben "Global Objectives existiert nicht wenn das Settlement System aktiviert ist". Es ist aber deaktiviert also müsste es doch existieren?

**Comment by D3nnis3n** [comment]
Ja, das ist richtig, sind aber zwei unterschiedliche Dinge.

Serverseitig:
a) Wenn Settlements aktiv sind, existiert das Feature nicht - die Berechtigung kann nicht gesetzt werden, sie erscheint aber fehlerhafterweise noch im Tooltip eines Titels, wenn der die Berechtigung noch hat, weil der Titel etwa mit einem Mod importiert wurde.
b) Wenn Settlements nicht aktiv sind, existiert das Feature und die die Berechtigung kann gesetzt werden.

Hier ist ein anderes Problem vorliegend, clientseitig:
Das entsprechende Feld wurde aus dem Client entfernt, daher kein Spieler kann ein solches eintragen, auch wenn das Feature aktiv ist.

Stand jetzt wird das Feature im nächsten Minor Update ganz entfernt, da wir im Client keine Unterscheidungen für verschiedene Servereinstellungen vorhalten wollen - da gab es leider einfach eine Misskommunikation zwischen Client- und Server-Team als Update 10 gemacht wurde.

**Comment by D3nnis3n** [comment]
Das "Global Objective" wurde in Update 10.3 vollständig entfernt.
